src/init.cpp

```diff
@@ -421,10 +421,6 @@ void initSystem()
 		SDL_GL_SetAttribute(SDL_GL_STENCIL_SIZE, 0);
 	}
 
-	SDL_GL_SetAttribute(SDL_GL_ACCUM_RED_SIZE, 16);
-	SDL_GL_SetAttribute(SDL_GL_ACCUM_GREEN_SIZE, 16);
-	SDL_GL_SetAttribute(SDL_GL_ACCUM_BLUE_SIZE, 16);
```

Comment:
But this breaks motion blur support on X11. Perhaps it's better to test for accumulation buffer support, and if not supported don't request is and also don't call `updateScreenMB()` in cplusplus/CGraphics.cpp anymore.

Reply:
Agreed. I'll change this tomorrow.